File: TestNG/Activities/Activity1.java
package testNGActivities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class Activity1 {
    WebDriver driver;

    @Test
    public void example() {
        String title = driver.getTitle();
        System.out.println(title);
        Assert.assertEquals(title, "Training Support");
        driver.findElement(By.id("about-link")).click();
        String newTitle = driver.getTitle();
        System.out.println(newTitle);
        Assert.assertEquals(newTitle, "About Training Support");
    }

    @BeforeMethod
    public void setUp() {
        //Install GeckoDriver
        WebDriverManager.firefoxdriver().setup();
        //Initialize driver
        driver = new FirefoxDriver();
        //Open a page
        driver.get("https://www.training-support.net/");
    }

    @AfterMethod
    public void tearDown() {
        driver.close();
    }
}

File: Selenium/Project/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Initialize wait object
    wait = WebDriverWait(driver, 10)

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms")

    print("Page title is: ", driver.title)

    assert driver.title == "Alchemy LMS – An LMS Application"